🐛 Bug: "Line is expected to be present" links no longer auto-fill info

### Bug Report Checklist

- [x] I am using the latest published version of all Flint packages.
- [x] I have [searched for related issues](https://github.com/flint-fyi/flint/issues?q=is%3Aissue) and found none that matched my issue.

### Expected

The `Flint bug: Line is expected to be present by the loop condition. Please report it here: ...` links should populate issue templates nicely.


### Actual

As seen in #2593 -> https://github.com/flint-fyi/flint/actions/runs/24937500506/job/73025479138?pr=2593:

```plaintext
Error: FlintAssertionError: Flint bug: Line is expected to be present by the loop condition. Please report it here: https://github.com/flint-fyi/flint/issues/new?template=03-general-bug.yaml&title=%F0%9F%90%9B+Bug%3A+Line+is+expected+to+be+present+by+the+loop+condition&actual=Assertion+message%3A%0A%0A%60%60%60%0ALine+is+expected+to+be+present+by+the+loop+condition%0A%60%60%60%0A%0AStacktrace%3A%0A%0A%60%60%60%0AError%3A+Flint+bug%3A+Line+is+expected+to+be+present+by+the+loop+condition.%0A++++at+assert+%28%3Ccensored+filename%3E%29%0A++++at+nullThrows+%28%3Ccensored+filename%3E%29%0A++++at+createReportSnapshotAt+%28%3Ccensored+filename%3E%29%0A++++at+createReportSnapshot+%28%3Ccensored+filename%3E%29%0A++++at+%3Ccensored+filename%3E%0A++++at+processTicksAndRejections+%28node%3Ainternal%3Ccensored+filename%3E%29%0A++++at+file%3Anode_modules%2F%40vitest%2Frunner%2Fdist%2Findex.js%3A1258%3A20%0A%60%60%60&additional_info=Process+arguments%3A%0A%0A%60%3Cnone%3E%60
```

<img width="1062" height="374" alt="Screenshot of a Flint blank new issue with the title filled out and blank body" src="https://github.com/user-attachments/assets/b25db1ea-dca9-4ed6-8d01-0e82db9ab5a6" />

### Additional Info

I think I broke this in #2475, heh.

❤️‍🔥

--- a/.changeset/great-dodos-wink.md
+++ b/.changeset/great-dodos-wink.md
@@ -1,0 +1,5 @@
+---
+"@flint.fyi/utils": patch
+---
+
+Correct general-bug.yaml number from 03 to 04.

--- a/packages/utils/src/assert.ts
+++ b/packages/utils/src/assert.ts
@@ -40,7 +40,7 @@ export function sanitizeStackTrace(stack: string): string {
 
 function buildIssueUrl(message: string, stack: string | undefined): string {
 	const issueUrl = new URL("https://github.com/flint-fyi/flint/issues/new");
-	issueUrl.searchParams.set("template", "03-general-bug.yaml");
+	issueUrl.searchParams.set("template", "04-general-bug.yaml");
 
 	const sanitizedStack = stack
 		? sanitizeStackTrace(stack)

